src/csharp/Grpc.Tools/build/_protobuf/Google.Protobuf.Tools.targets

```diff
@@ -68,9 +68,9 @@
 
     <PropertyGroup>
       <!-- First try environment variable. -->
```

Comment:
Comment should be updated to say that environment variables don't override explicitly set values

Reply:
Great catch! Would be good to know what is the intended behavior from @kkm000 

Do we want env variables to overwrite props or do we want props to overwrite env? Dependent on that, we can also only overwrite the prop if env is not empty. This would also fix the bug of not being able to set it at all right now.

I‘m not sure if the use cases here. Maybe having it set in env for build servers? Then env should overwrite props.

Reply:
Maybe @jtattermusch also knows if we want to have ENV overwrite PROPS or the other way around to adjust the logic accordingly.